code/modules/admin/verbs/listahelps.dm

```diff
@@ -0,0 +1,318 @@
+/datum/adminticket
+	var/ID = ""
+	var/user = ""
+	var/uckey
+	var/admin = "N/A"
+	var/msg = ""
+	var/resolved = "No"
+	var/permckey = ""
+	var/permuser = ""
+	var/uID = ""
+	var/active = "No"
+	var/logs = list()
+	var/replying = 0
```

Comment:
instead of using 0, use the defines TRUE and FALSE which are, respectively, 1 and 0